server/bin/basket-proxy-server.js

```diff
@@ -150,8 +150,8 @@ function initApp() {
     basketRequest('/lookup-user/?email=' + email, 'get', params)
       .on('error', function (error) {
         logger.error('lookup-user.error', error);
-      })
-      .pipe(res);
+      });
+      //.pipe(res);
   });
```

Comment:
This is to check if the error is thrown when basket does not return anything. Will change back once we arrive at a solution.